NEWS.md

```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```

Comment:
FWIW, any contributor is able to create or edit a release draft on github at any time to keep track of release notes ahead of an actual release; e.g. with dgrid I try to keep a draft updated as each thing gets merged. Just have to be careful not to prematurely click the big green button on the page (why do they have to always have tempting big green buttons?!)

Not sure if that influences your thought of doing both a NEWS.md and github release notes.